index.js

```diff
@@ -0,0 +1,65 @@
+const books = [];
+const title = document.querySelector('.title');
+const author = document.querySelector('.author');
+const displayContainer = document.querySelector('.container');
+const storage = () => {
+  localStorage.setItem('books', JSON.stringify(books));
+};
+const addBtn = document.querySelector('.add');
+addBtn.addEventListener('click', (event) => {
+  event.preventDefault();
+  if (title.value !== '' && author.value !== '') {
+    const book = {
+      title,
+      author,
+    };
+    books.push(book);
```

Comment:
- [ ] Let's create a function that adds a new book to the collection, with title and author. It makes your code more readable and improves the separation of tasks.